server/routes/manage-csips/view.njk

```diff
@@ -103,61 +111,47 @@
         text: csipRecord.referralDate | formatSimpleDisplayDate
       },
       {
-        html: reviewDate({
-          date: csipRecord.nextReviewDate | formatSimpleDisplayDate,
-          emptyLabel: '-'
-        })
+        classes: 'col-width-200',
+        html: csipRecord.logCode or '-'
       },
       {
         classes: 'col-width-200',
         html: (csipRecord.caseManager | softHyphenate(20)) or '-'
-      },
+      } if pageName != 'referrals',
```

Comment:
Perhaps consider moving these page name strings into constants at the top of this view (or elsewhere) so they're not repeated